types/evidence.go

```diff
@@ -554,6 +555,76 @@ func LightClientAttackEvidenceFromProto(lpb *tmproto.LightClientAttackEvidence)
 // EvidenceList is a list of Evidence. Evidences is not a word.
 type EvidenceList []Evidence
 
+// StringIndented returns a string representation of the evidence.
+func (data *EvidenceList) StringIndented(indent string) string {
```

Comment:
does this need to be a pointer reciever? Slices themselves are pointers (after a fashion,) so it might clear (some) things up if we drop the `*` from the receiver type.

Reply:
Cool, makes sense. Thanks!